hope.yaml

```diff
@@ -429,6 +445,13 @@ resources:
       - DOCKER_REGISTRY_HOSTNAME
       - DOCKER_CONFIG_JSON_FILE_CONTENTS_BASE64
     tags: [apps, registry]
+  - name: cluster-registry-secrets-tasks
```

Comment:
cluster-registry-secrets, certbot copy, job deleted, pod killer.

There are a lot of things that need to be duplicated across namespaces just to keep them running clean. Should find a way around all the duplication.

Maybe even include them all in a template file for deploying the more comprehensive namespaces.